examples/palace-info.py

```diff
@@ -37,6 +37,19 @@
 
     print(f'\nInfo of device {args.device.name!r}:')
     print('ALC version: {}.{}'.format(*args.device.alc_version))
+
+    # TODO: AL info
+    with Device() as dev, Context(dev) as ctx:
+        print()
+        print('Available resamplers:')
+        default_idx = ctx.default_resampler_index
+        default_resampler = ctx.available_resamplers[default_idx]
+        for resampler in ctx.available_resamplers:
+            if resampler == default_resampler:
```

Comment:
What is this lol.  Could you also please separate the processing part from the printing part, which I believe will enable you to write this more concisely.